lib/awful/rules.lua.in

```diff
@@ -193,37 +194,152 @@ function rules.apply(c)
     rules.execute(c, props, callbacks)
 end
 
+--- Get a property value
+-- @param c The client.
+-- @param property The property name
+-- @param value The property value
+-- @param ignore_function A table of properties where functions should not be executed
```

Comment:
What is `ignore_function` required for?
Wouldn't it be easier to not have this extra list, but check for type == "function" (instead of ~= "function")?